tmail-backend/tmail-third-party/ai-bot/src/test/java/com/linagora/tmail/mailet/AIBotMailetTest.java

```diff
@@ -48,48 +51,35 @@ public static MailAddress createMailAddress(String mailAddress) {
 
     public static final String DEMO_MODEL = "gpt-4o-mini";
     private static final MailAddress ASKING_SENDER = createMailAddress("sender@example.com");
-    private static final MailAddress BOT_ADDRESS = createMailAddress("gpt@example.com");
+    private static final MailAddress BOT_ADDRESS = createMailAddress("gpt@localhost");
 
-    private AIBotMailet testee;
     private MailetContext mailetContext;
+    private Configuration configuration;
+    private AIBotConfig aiBotConfig;
+    private AIBotMailet testee;
 
     @BeforeEach
     void setUp() {
-        testee = new AIBotMailet(new ChatLanguageModelFactory(), new JsoupHtmlTextExtractor());
+        configuration = new PropertiesConfiguration();
+        configuration.addProperty("apiKey", "demo");
+        configuration.addProperty("botAddress", "gpt@localhost");
+        configuration.addProperty("model", "gemini-2.0-flash");
+        configuration.addProperty("baseURL", "https://generativelanguage.googleapis.exemple.com");
+        aiBotConfig= AIBotConfig.fromMailetConfig(configuration);
```

Comment:
Build a AIconfig directly we do not need to go through Configuration that needs extra parsing steps.
Let's be more straightforward.